meta-lmp-bsp/recipes-security/optee/optee-os-fio-mfgtool_3.10.0.bbappend

```diff
@@ -14,6 +14,12 @@ EXTRA_OEMAKE_append_apalis-imx6 = " \
     CFG_TZDRAM_START=0x4e000000 CFG_OVERLAY_ADDR=0x16000000 \
     CFG_OVERLAY_RESERVED_MEMORY_ADDRESS_CELLS=1 CFG_OVERLAY_RESERVED_MEMORY_SIZE_CELLS=1 \
 "
+EXTRA_OEMAKE_append_apalis-imx6-sec = " \
+    CFG_NXP_CAAM=n CFG_IMX_CAAM=y \
```

Comment:
Shouldn't we change for apalis-imx6 instead? The original issue reported would both cases, right?

Reply:
done